stream_test.go

```diff
@@ -52,6 +53,18 @@ var _ = Describe("Stream", func() {
 		resetCalledAtOffset = offset
 	}
 
+	// streamWithTimeout creates an io.ReadWriter that makes sure that Read and Write time out
+	streamWithTimeout := func(str io.ReadWriter) io.ReadWriter {
+		timeout := scaleDuration(250 * time.Millisecond)
```

Comment:
Do we really need to do this everywhere? Why not just pick 1s, it shouldn't fail that often locally, right? I though this was only an issue for the deadlines…